### PR TITLE
e2e: add hello_world test image source

### PR DESCRIPTION
**Pull request by mythi**

### CODE REVIEW

**Review by Xynnn007 — APPROVED** [review]
Thanks for sharing this!

**Comment by mythi** [comment]
Let's keep this open until #140 is done.

File: tools/packaging/build/test-image/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:20.04 as builder
+
+RUN apt-get update && \
+    env DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+    build-essential
+
+COPY hello.c .
+
```
Comment (Xynnn007) [inline tools/packaging/build/test-image/Dockerfile:8]: To make the decryption key and decryption key id more clear, I added two lines here
```
LABEL enc_key="LieOhvkqFcGMzZrVzt6vPWlj/F/bgYMNe45vhQpdxAA="

LABEL key_id="kbs:///default/image-kek/11032d96-dccd-46a3-9244-93644d76745f"
```

Comment (mythi) [reply tools/packaging/build/test-image/Dockerfile:8]: These will likely trigger Github secret scanning alarms. Why do we need them?

Comment (Xynnn007) [reply tools/packaging/build/test-image/Dockerfile:8]: These are test images and will be useful to know which key & keyid used for better maintance. 
We can use `skopeo inspect ghcr.io/confidential-containers/test-container-enclave-cc:encrypt` to check the information like
```
root@xynnn:/home/enclave-cc/tools/packaging/build/test-image# skopeo inspect docker://ghcr.io/confidential-containers/test-container-enclave-cc:encrypted
{
    "Name": "ghcr.io/confidential-containers/test-container-enclave-cc",
    "Digest": "sha256:8e89aead9b7650beb12fde04fad063342bf559efb42b51849484a338f46e21c0",
    "RepoTags": [
        "unencrypted",
        "encrypted",
        "sample_kbc_encrypted"
    ],
    "Created": "2023-04-02T18:46:57.024855281+08:00",
    "DockerVersion": "",
    "Labels": {
        "enc_key": "LieOhvkqFcGMzZrVzt6vPWlj/F/bgYMNe45vhQpdxAA=",
        "key_id": "kbs:///default/image-kek/11032d96-dccd-46a3-9244-93644d76745f"
...
```

This is the same as kata-cc test images.

And sorry the two LABELs should be placed under the line `FROM scratch`, not where this comment is at. It is
```
FROM ubuntu:20.04 as builder

RUN apt-get update && \
    env DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
    build-essential

COPY hello.c .

RUN gcc hello.c -fPIE -pie -o hello_world

FROM scratch

LABEL enc_key="LieOhvkqFcGMzZrVzt6vPWlj/F/bgYMNe45vhQpdxAA="

LABEL key_id="kbs:///default/image-kek/11032d96-dccd-46a3-9244-93644d76745f"

COPY --from=builder /hello_world /bin/
```

Comment (mythi) [reply tools/packaging/build/test-image/Dockerfile:8]: Pushed updates

**Review by Xynnn007 — APPROVED** [review]

**Pull request merged**